Tests/UnitTests/ReaderTests/Data/ImageReaderSimpleRegression_map.txt

```diff
@@ -0,0 +1,4 @@
+images\11.png	1	1
```

Comment:
Just saw that I should add one for training AND testing. The current test is doing it for training and writing, only. I'll add a test run, no problem. May take another hour or so.